src/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobReader.cs

```diff
@@ -435,10 +435,11 @@ private static void ThrowInvalidSerializedString()
         /// <summary>
         /// Reads type code encoded in a serialized custom attribute value. 
         /// </summary>
+        /// <returns><see cref="SerializationTypeCode.Invalid"/> if the encoding is invalid.</returns>
```

Comment:
If we have a <returns/> comment, we probably want it to say what it returns not just in the case of error, right?